d2renderers/d2svg/d2svg.go

```diff
@@ -1632,7 +1633,7 @@ const DEFAULT_THEME int64 = 0
 
 var DEFAULT_DARK_THEME *int64 = nil // no theme selected
 
-func Render(diagram *d2target.Diagram, opts *RenderOpts) ([]byte, error) {
+func Render(diagram *d2target.Diagram, g *d2graph.Graph, opts *RenderOpts) ([]byte, error) {
```

Comment:
oh, nah we shouldn't be passing `g` into Render. i guess you will have to redefine the getUniqueChars again on `d2target.Diagram`

Reply:
oh the link and tooltip would appear on appendix, so you do still need those in the `d2target.Diagram` version of getUniqueChars